scripts/README.md

```diff
@@ -14,46 +14,83 @@ npx truffle compile
+
+In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. _Otherwise, everyone can steal the funds from your account!_
+
+Setup env variables for the deployment process:
 
 ```
 export PK=<Your Key>
-export GAS_PRICE_GWEI=8
+export GAS_PRICE_GWEI=<look up the suggestion from ethgasstation.info>
 export NETWORK_NAME=<network>
 export MASTER_SAFE=<master safe>
 
 ```
 
-### Confirming multisig orders with Metamask
+### Confirming multisig-transactions on gnosis-safe with Metamask
+
+For the signing process, note that the gas consumption is underestimated. There is currently no proper way to estimate the gas limit. In order to make sure that the gas limit of your transaction is sufficient, increase it to 5 million gas. In practice, this limit suffices to deploy a strategy with 20 brackets.
+
+### Deploy the bracket-strategy:
 
-These script require to set up a "Master Safe" created through the interface (here)[https://rinkeby.gnosis-safe.io] with an owner key under your control. This Master Safe should have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface and this implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
-In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. Otherwise, everyone can steal the funds from your account!
+In order to deploy new bracket-trader contracts, place orders on behalf of the newly mined contracts and fund their accounts on the exchange, one only has to run the `complete_liquidity_provision` script.
+It will create one ethereum transaction for creating the safes, and send two transactions request to the gnosis-safe interface.
 
-For the signing process, note that Metamask underestimates gas consumption. Just make sure that the gas limit of your transaction is sufficient. There is currently no proper way to estimate the gas limit. In practice, setting the gas limit to a value higher than 5,000,000 for transactions involving 20 brackets is a practical, yet not fully reliable way to ensure a successful transaction.
+The first request will generate orders on behalf of the bracket-traders.
+Please sign this transaction in the gnosis-safe interface and double check that the prices of the orders are as intended -for example in [telegram-mainnet](https://t.me/gnosis_protocol) or [telegram-rinkeby](https://t.me/gnosis_protocol_dev) channels.
+Then continue the script and send out the second request, which will fund the bracket-traders' accounts on the exchange.
```

Comment:
"Then" should be "after the transaction appears as confirmed in the web interface", otherwise the user creates two transactions with the same nonce and the second will automatically fail.